languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/replace.txt

```diff
@@ -482,4 +482,7 @@ Goncalo Ramos=Gonçalo Ramos
 Ousmane Dembele=Ousmane Dembélé
 Luka Jovic=Luka Jović
 Türkisch Airline=Turkish Airlines
-Türkisch Airlines=Turkish Airlines
\ No newline at end of file
+Türkisch Airlines=Turkish Airlines
+Finanztip=Finanztipp	Wenn Sie nicht den Ratgeber "Finanztip" meinen, wird "Finanztipp" mit doppeltem "p" geschrieben.
+Vermögensteuer=Vermögenssteuer
```

Comment:
Beides ist aber korrekt, oder? Dann vielleicht mit Kommentar, dass wir eine Variante empfehlen?

Reply:
Ja, beides ist korrekt, wobei letzteres viermal mehr verwendet wird. Aber ja, dann packe ich es wohl lieber in die recommendation.csv ...

Reply:
Vermögensteuer ist die korrekte Schreibweise im juristischen Bereich. https://www.gesetze-im-internet.de/vstg_1974/BJNR109490974.html
Gilt meines Wissens für alle Komposita mit -steuer. Siehe hierzu auch https://de.wikipedia.org/wiki/Fugenlaut unter "Weitere Aspekte der Verwendung von Fugenelementen".
Schaden(s)ersatz ist anscheinend ein Streitthema. Mein Mann, der Richter, besteht auf Schadenersatz. Aber der Gesetzgeber schreibt in BGB inzwischen klar Schaden**s**ersatz (https://www.gesetze-im-internet.de/bgb/__280.html)
Die Justiz in Hamburg verwendet übrigens LanguageTool. ;-)